src/System.Management.Automation/security/SecureStringHelper.cs

```diff
@@ -456,13 +458,14 @@ public static byte[] Protect(byte[] userData, byte[] optionalEntropy, DataProtec
                     dwFlags |= CAPI.CRYPTPROTECT_LOCAL_MACHINE;
                 unsafe
                 {
-                    if (!CAPI.CryptProtectData(new IntPtr(&dataIn),
-                                                string.Empty,
-                                                new IntPtr(&entropy),
-                                                IntPtr.Zero,
-                                                IntPtr.Zero,
-                                                dwFlags,
-                                                new IntPtr(&blob)))
+                    if (!CAPI.CryptProtectData(
+                        new IntPtr(&dataIn),
+                        string.Empty,
+                        new IntPtr(&entropy),
+                        IntPtr.Zero,
+                        IntPtr.Zero,
+                        dwFlags,
+                        new IntPtr(&blob)))
```

Comment:
Might be a good idea to name arguments here for readability, especially the zero pointers and empty string.